packages/editor-ui/src/views/NodeView.vue

```diff
@@ -4197,68 +4191,43 @@ export default defineComponent({
 				workflow_id: this.workflowsStore.workflowId,
 			});
 		},
-		onAddNode(
-			nodeTypes: Array<{ nodeTypeName: string; position: XYPosition }>,
-			dragAndDrop: boolean,
+		async onAddNodes(
```

Comment:
can you please pull in master? lots of changes were merged in related to the AI work that might create conflicts for you?

Reply:
Already did, fixed the conflicts 😄